BUG: fix unexpected return of np.pad with mode=wrap (#22575)
numpy/numpy: doc/release/upcoming_changes/22575.compatibility.rst

> ``np.pad`` with ``mode=wrap`` now returns expected results
> ----------------------------------------------------------

Reviewer: ```suggestion
``np.pad`` with ``mode=wrap`` pads with strict multiples of original data
-------------------------------------------------------------------------
```

> Codes based on earlier version of ``pad`` with ``wrap`` will 
> break when padding size is larger than initial array.

Reviewer: ```suggestion
Code based on earlier version of ``pad`` that uses  ``mode="wrap"`` will return
different results when the padding size is larger than initial array.
```
I think together with the following sentence, this explains more precisely how the code will break.

``np.pad`` with ``mode=wrap`` now always fills the space with 

> strict loops of original data, as expected.

Reviewer: ```suggestion
strict multiples of original data even if the padding size is larger than the
initial array.
```
"multiples" might be clearer to someone just skimming this.
